app/src/main/res/values/strings.xml

```diff
@@ -386,13 +386,13 @@
 
     <!-- cert4android -->
     <string name="certificate_notification_connection_security">EteSync: Connection security</string>
-    <string name="trust_certificate_unknown_certificate_found">EteSync has encountered an unknown certificate. Do you want to trust it?</string>
+    <string name="trust_certificate_unknown_certificate_found">An unknown certificate was received. Is it a certificate you have confirmed your trust of?</string>
```

Comment:
Should be "in", but maybe I can think of a way to avoid it being quirky while still having all these elements.

Reply:
It feels very convoluted indeed. How about:
"An unknown certificate was received, would you like to trust it?"